### PR TITLE
#2239 remove banner landmark from promo banner

### PR DESCRIPTION
**Pull request by tjheffner**
## Chromatic

https://2239-remove-banner-landmark--60f9b557105290003b387cd5.chromatic.com

---

## Description
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2239. 

The ticket describes:

> The <va-promo-banner> currently has a role="banner" attribute, which is not correct. The banner role is intended for the global header (the <header> is the semantic equivalent of this role).

We were also seeing some errors reported in the next-build repo related to duplicate banner roles existing on the page (because `<header role="banner">` exists already)
Screenshot 2023-11-14 at 2 19 45 PM

## Testing done
I removed `role="banner"` from the `va-promo-banner` component and its related tests.

in `web-components/` I ran `yarn test`:

```
 PASS  src/components/va-promo-banner/test/va-promo-banner.e2e.ts
```

I also confirmed in Storybook locally.

## Screenshots
Running storybook locally, verified that the banner still rendered and did not have the role present.
Screenshot 2023-11-15 at 11 00 11 AM

## Acceptance criteria
- [x] The `<va-promo-banner>` component renders without a banner landmark.

## Definition of done
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "32.0.0",
+  "version": "32.0.1",
```
Comment (jamigibbs) [inline packages/core/package.json:4]: Hi @tjheffner thank you for this update! Do you mind reverting this `core` version change? The design system team will handle updating this when a new component library release is made.

Comment (tjheffner) [reply packages/core/package.json:4]: Not at all! There were some notes in the PR template so I assumed I had to do it for it to go live. Will revert.

**Review by jamigibbs — APPROVED** [review]
Thanks again for helping out! You could have left the version update in the `web-components` package.json but not a big deal. We use `core` to create releases for the packages (which is what DST will handle).

Approved! 👍

**Comment by jamigibbs** [comment]
@tjheffner I'm going to merge this on your behalf because it looks like you might be out on PTO. We will be creating a release today or tomorrow.

**Pull request merged**